src/test/kotlin/nomic/api/RulesAmendmentsTests.kt

```diff
@@ -0,0 +1,35 @@
+package nomic.api
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.web.client.TestRestTemplate
+import org.springframework.boot.test.web.client.getForEntity
+import org.springframework.http.HttpStatus
+
+@SpringBootTest(
+    webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT
+)
+class RulesAmendmentsTests(@Autowired val client: TestRestTemplate) {
```

Comment:
Yeah...not going to write a test for that.  It's overkill

Reply:
It would probably at least be worth changing the mock data then to ensure that more than one amendments are getting attached and that there aren't any weird bugs where only the first amendment is getting attached.

Reply:
Any reason why this wouldn't be a worthwhile test in case of software regression (detecting which is one of the driving motivation behind testing - otherwise manually testing the first time would suffice)